README.md

```diff
@@ -1,11 +1,8 @@
-# Welcome to GSTools
+# Welcome to GSTools-Cython
 [![GMD](https://img.shields.io/badge/GMD-10.5194%2Fgmd--15--3161--2022-orange)](https://doi.org/10.5194/gmd-15-3161-2022)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1313628.svg)](https://doi.org/10.5281/zenodo.1313628)
-[![PyPI version](https://badge.fury.io/py/gstools.svg)](https://badge.fury.io/py/gstools)
-[![Conda Version](https://img.shields.io/conda/vn/conda-forge/gstools.svg)](https://anaconda.org/conda-forge/gstools)
-[![Build Status](https://github.com/GeoStat-Framework/GSTools/workflows/Continuous%20Integration/badge.svg?branch=main)](https://github.com/GeoStat-Framework/GSTools/actions)
-[![Coverage Status](https://coveralls.io/repos/github/GeoStat-Framework/GSTools/badge.svg?branch=main)](https://coveralls.io/github/GeoStat-Framework/GSTools?branch=main)
-[![Documentation Status](https://readthedocs.org/projects/gstools/badge/?version=latest)](https://geostat-framework.readthedocs.io/projects/gstools/en/stable/?badge=stable)
+[![Build Status](https://github.com/GeoStat-Framework/GSTools-Cython/workflows/Continuous%20Integration/badge.svg?branch=main)](https://github.com/GeoStat-Framework/GSTools-Cython/actions)
+[![Coverage Status](https://coveralls.io/repos/github/GeoStat-Framework/GSTools-Cython/badge.svg?branch=main)](https://coveralls.io/github/GeoStat-Framework/GSTools-Cython?branch=main)
+[![Documentation Status](https://readthedocs.org/projects/gstools-cython/badge/?version=latest)](https://geostat-framework.readthedocs.io/projects/gstools-cython/en/stable/?badge=stable)
```

Comment:
Do you think we really need a readthedocs-site for GSTools-Core?

Reply:
I set it up and it is cool to have a documentation for the low level routines. So it is there, why should we remove it again?

Reply:
Will keep it.